bfcfg

```diff
@@ -687,9 +714,16 @@ sys_cfg()
   sys_cfg_one_byte ${tmp_file} "ENABLE_SPCR" 32 "${SYS_ENABLE_SPCR}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_PCIE" 33 "${SYS_DISABLE_PCIE}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_OPTEE" 34 "${SYS_ENABLE_OPTEE}"
+  sys_cfg_one_byte ${tmp_file} "DISABLE_TMFF" 35 "${SYS_DISABLE_TMFF}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_I2C0" 36 "${SYS_ENABLE_I2C0}"
   sys_cfg_one_byte ${tmp_file} "DISABLE_FORCE_PXE_RETRY" 37 "${SYS_DISABLE_FORCE_PXE_RETRY}"
   sys_cfg_one_byte ${tmp_file} "ENABLE_BMC_FIELD_MODE" 38 "${SYS_ENABLE_BMC_FIELD_MODE}"
+  sys_cfg_one_byte ${tmp_file} "DEFAULT_LARGE_ICMC_LOCK" 43 "${SYS_DEFAULT_LARGE_ICMC_LOCK}"
+  sys_cfg_one_byte ${tmp_file} "DISPLAY_FLAGS" 44 "${SYS_DISPLAY_FLAGS}"
```

Comment:
This isn't a real platform configuration; this is toggled through key chain. I don't think we should expose this option.